extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -79,13 +79,14 @@ class AmpFitText extends AMP.BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
+  isRelayoutNeeded() {
     return true;
   }
 
   /** @override */
-  isRelayoutNeeded() {
-    return true;
+  prerenderCallback() {
```

Comment:
Actually, I prefer to do the other way around, call `prerenderCallback` from inside `layoutCallback`. Just in case `layoutCallback` changes in the future to be more costly.